readme.md

```diff
@@ -9,27 +9,6 @@ With NodeJS installed, just run NPM:
 
 > npm install can-compile -g
 
-## Command line
```

Comment:
Yes please. I think it is important that it works standalone (I personally barely use any build tools anymore because most tools now have a great CLI).